### PR TITLE
Close some rel management issues, update Versioning to latest behavior

### PR DESCRIPTION
**Pull request by marcelolr**

### CODE REVIEW

File: RELEASE-MANAGEMENT.md [hunk 1]
```diff
@@ -2,22 +2,14 @@
...
-ISSUE: How do we use experimental on operators (or potentially IR features) for a given release/pre-release?
-
-ISSUE: How do we communicate schedules and exit criteria for a given pre-release/release?
-
-ISSUE: How do we stablize one pre-release/release while we are working on subsequent ones?
+Releases
+--------
 
-ISSUE: which files should be included? Probably an archive of all files (minus tests if they get large). How about protoc targets?
+Releases are versioned according to docs/Versioning.md. This describes IR and operator versioning policies, as well as a propose how models themselves should be versioned.
```
Comment (prasanthpul) [inline RELEASE-MANAGEMENT.md]: Can you make "docs/versioning.md" a hyperlink?

"as well as a propose how models" => some words missing or some extra words

File: docs/Versioning.md [hunk 2]
```diff
@@ -90,9 +77,12 @@ ONNX specification. The union of the operator sets specified by a given model MU
 
 How nodes bind to operator declarations is strictly defined and are designed to increase model compatibilitey across ONNX implementations (appealing to the conservative clause of the robustnes principle). 
 
-How ONNX implementations bind an operator declaration to specific implementation is outside the scope
-of this specification.  Implementations of ONNX MAY elect 
-to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle. 
+How ONNX implementations bind an operator declaration to specific implementation is outside the scope of this specification.
+Implementations of ONNX MAY elect to introduce more sophisticated operator declaration/implementation binding modes to appeal to the liberal clause of the robustness principle.
+
+Implementations and models MAY provide experimental operators by agreeing to use operators in an experimental domain.
```
Comment (prasanthpul) [inline docs/Versioning.md]: there is no experimental domain. it's just a flag on the regular domain.

File: docs/Versioning.md [hunk 3]
```diff
@@ -90,9 +77,12 @@ ONNX specification. The union of the operator sets specified by a given model MU
...
+
+Implementations and models MAY provide experimental IR features by agreeing to use fields from reserved ranges.
```
Comment (prasanthpul) [inline docs/Versioning.md]: Not sure what this means in practice. Experimental ops are documented with the regular ops. where are these experimental fields documented? How do we avoid conflicts between different experimenters?

File: RELEASE-MANAGEMENT.md (hunk 1 above)
Comment (marcelolr) [reply RELEASE-MANAGEMENT.md]: fixing

File: docs/Versioning.md (hunk 2 above)
Comment (marcelolr) [reply docs/Versioning.md]: Well, the could do that ;) Sorry, these are really about extensions not versioning, I've started a section over in IR and they belong there.

File: docs/Versioning.md (hunk 3 above)
Comment (marcelolr) [reply docs/Versioning.md]: See above. I mixed versioning with extending here, I'm removing that verbiage.

**Pull request merged**